### ISSUE TITLE
Rows with empty referenceUid should be ignored

### ISSUE THREAD

**Issue by glucka** [cobwebch/external_import#241]
Currently, rows where referenceUid is empty will also be imported. 
I think this kind of rows could be ignored.

**Comment by glucka** [comment]
#36

**Comment by glucka** [comment]
https://github.com/cobwebch/external_import/issues/36
I see that you suggested the solution with custom import steps, 
but i think we should filter the empty referenceUid-s as default.

**Comment by fsuter** [comment]
I agree. Importing rows with an empty referenceUid does not make sense. I'll integrate that in a future release.
